pkg/kubelet/kubelet.go

```diff
@@ -33,7 +33,7 @@ import (
 	"time"
 
 	cadvisorapi "github.com/google/cadvisor/info/v1"
-	v1 "k8s.io/api/core/v1"
```

Comment:
Why is this needed?

Reply:
My local `gofmt` seems to have done those automagically. Undid these changes

Reply:
If `hack/update-gofmt.sh` makes the changes that's OK.  But you shouldn't use your own `gofmt`.